README.md

```diff
@@ -211,17 +211,16 @@ Run `make ocm/install/rhmi-addon` to trigger the installation. Once the installa
 
 ## Release
 
-Update the operator version in the following files:
-
-* Update `TAG` and `PREVIOUS_TAG` (the previous version) in the [Makefile](Makefile)
-
-* Generate a new CSV:
-```sh
-make gen/csv
-```
-
-Commit changes and open pull request. When the PR is accepted, create a new release tag:
-
-```sh
-git tag v<version> && git push upstream v<version>
-```
+- Create `release-<release-number>` branch
+- Update `tag` and `previoustag` in makefile
+- Run `make gen/csv`
+- - Perform any manual tidying up of the generated CSV as required.
```

Comment:
What manual tidying up is needed?

Reply:
well, it depends, I suppose this could be phrased as "sanity check the result" but there is quite often manual changes required, though they are different each time.

Reply:
this would block automation. Do we have some examples? Should they be bugs in the operator sdk